docs/pipeline-components-and-applications/iglu/igluctl-2/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Igluctl"
-date: "2021-01-21"
+date: "2023-04-26"
```

Comment:
You can just remove this, Docusaurus ignores it (older pages have the dates because the migration from Wordpress added them)